src/loaders/HubsTextureLoader.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * @author mrdoob / http://mrdoob.com/
+ */
```

Comment:
?

Reply:
We should probably just add the threejs MIT header instead? HubsTextureLoader is a modified THREE.TextureLoader.